Assets/Thirdweb/Core/Scripts/Bridge.cs

```diff
@@ -27,6 +27,8 @@ public RequestMessageBody(string[] arguments)
         }
 
         private static Dictionary<string, TaskCompletionSource<string>> taskMap = new Dictionary<string, TaskCompletionSource<string>>();
+        private static Dictionary<string, Delegate> taskActionMap = new Dictionary<string, Delegate>();
+        private static Dictionary<string, Type> taskActionTypeMap = new Dictionary<string, Type>();
```

Comment:
Would it make sense to have a map from task id to struct { delegate, type } instead of two maps?